src/hooks/useHasFocus/useHasFocus.test.tsx

```diff
@@ -15,7 +16,7 @@ describe('useHasFocus', () => {
 
     const result = render(<TestComponent />);
 
-    expect(result.queryByTestId('focus')).not.toBeInTheDocument();
+    expect(document.contains(result.queryByTestId('focus'))).not.toBeTruthy();
```

Comment:
I'm not a big fan of going back to basic assertions.

Dom assertions are really useful, especially for giving more meaningful error messages in your test ("element not in document," vs "true not equals false").

Reply:
We can add `@testing-library/jest-dom`, it's all compatible with the `chai` API, but it will mess with the global types because it adds all global `jest` functions (`expect`, `it`, `describe` etc.).

We use the `toBeInTheDocument` matcher just once in the project so it's not worth adding it (or an alternative) to the project now. We don't use any other matchers from `@testing-library/jest-dom` either.